sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UserProfileVisibilityTest.java

```diff
@@ -0,0 +1,120 @@
+package org.sunbird.integration.test.user;
+
+import com.consol.citrus.annotations.CitrusTest;
+import com.consol.citrus.testng.CitrusParameters;
+import javax.ws.rs.core.MediaType;
+import org.springframework.http.HttpStatus;
+import org.sunbird.common.action.TestActionUtil;
+import org.sunbird.common.action.UserUtil;
+import org.sunbird.integration.test.common.BaseCitrusTestRunner;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class UserProfileVisibilityTest extends BaseCitrusTestRunner {
+  public static final String TEST_USER_PROFILE_VISIBILITY_FAILURE_WITHOUT_ACCESS_TOKEN =
+      "testUserProfileVisibilityFailureWithoutAccessToken";
+
+  public static final String TEST_USER_PROFILE_VISIBILITY_FAILURE_WITH_INVALID_USERID =
+      "testUserProfileVisibilityFailureWithInvalidUserId";
+  public static final String TEST_USER_PROFILE_VISIBILITY_SUCCESS_WITH_VALID_USERID =
+      "testUserProfileVisibilitySuccessWithValidUserId";
+  public static final String TEST_USER_PROFILE_VISIBILITY_FAILURE_WITH_INVALID_COLUMN =
+      "testUserProfileVisibilityFailureWithInvalidColumn";
+  public static final String
+      TEST_USER_PROFILE_VISIBILITY_FAILURE_WITH_DUPLICATE_COLUMN_IN_PRIVATE_AND_PUBLIC_ARRAY =
+          "testUserProfileVisibilityFailureWithDuplicateColumnInPrivateAndPublicArray";
```

Comment:
Have two tests for this scenario.

testUserProfileVisibilityFailureWithDuplicatePrivateColumns
testUserProfileVisibilityFailureWithDuplicatePublicColumns